web-server/v0.4/tests/fix_puppeteer.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Do we use this file?

Reply:
This is also used for testing. This will be removed under this PR.